src/CoreBundle/EventListener/AddJqueryIncludeListener.php

```diff
@@ -21,18 +21,22 @@ class AddJqueryIncludeListener
      */
     public function onGlobalHtmlHeaderInclude(HtmlIncludeEventInterface $event)
     {
+        $jqueryInclude = '';
         if (true === TGlobal::IsCMSMode()) {
-            $sInclude = '<script src="'.TGlobal::GetStaticURL('/chameleon/blackbox/javascript/jquery/jquery-3.3.1.min.js').'" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->';
-            $sInclude .= '<script src="'.TGlobal::GetStaticURL('/chameleon/blackbox/javascript/jquery/jquery-migrate-1.4.1.js').'" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->';
+            $jqueryInclude = '<script src="'.TGlobal::GetStaticURL('/chameleon/blackbox/javascript/jquery/jquery-3.3.1.min.js').'" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->';
+            $jqueryInclude .= '<script src="'.TGlobal::GetStaticURL('/chameleon/blackbox/javascript/jquery/jquery-migrate-1.4.1.js').'" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->';
         } else {
             if (defined('CHAMELEON_URL_GOOGLE_JQUERY') && CHAMELEON_URL_GOOGLE_JQUERY !== false) {
-                $sInclude = '<script src="'.CHAMELEON_URL_GOOGLE_JQUERY.'"></script>';
+                $jqueryInclude = '<script src="'.CHAMELEON_URL_GOOGLE_JQUERY.'"></script>';
                 $sJQueryLocal = '<script src="'.TGlobal::GetStaticURL(CHAMELEON_URL_JQUERY).'" type="text/javascript"></script>';
-                $sInclude .= "\n<script>window.jQuery || document.write('".str_replace('/', '\\/', $sJQueryLocal)."')</script>";
-            } else {
-                $sInclude = '<script src="'.TGlobal::GetStaticURL(CHAMELEON_URL_JQUERY).'" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->';
+                $jqueryInclude .= "\n<script>window.jQuery || document.write('".str_replace('/', '\\/', $sJQueryLocal)."')</script>";
+            } else if (CHAMELEON_URL_JQUERY !== false) {
+                $jqueryInclude = '<script src="'.TGlobal::GetStaticURL(CHAMELEON_URL_JQUERY).'" type="text/javascript"></script><!--#GLOBALRESOURCECOLLECTION#-->';
             }
         }
-        $event->addData(array($sInclude));
+
+        if ('' !== $jqueryInclude) {
+            $event->addData([$jqueryInclude]);
+        }
```

Comment:
Instead of checking for an emtpy string, couldn't we just add the data in every branch of the if statements above? This would make it simpler to read in my opinion.

Reply:
Done